lib/huginn_acumen_product_agent/concerns/acumen_product_query_concern.rb

```diff
@@ -37,11 +37,20 @@ def get_product_contributors(acumen_client, products)
             id = product['acumenAttributes']['product_marketing_id']
             product_contributor = product_contributors[id]
 
+
             if product_contributor
-                product['contributor'] = product_contributor.map do |pc|
+                contributor_ids = product_contributor.map do |pc|
+                  pc['contributor_id']
+                end
+
+                type_response = acumen_client.get_contributor_types(contributor_ids)
+                contributor_types = process_contributor_types_query(type_response)
+
+                product['contributors'] = product_contributor.map do |pc|
                     {
                         '@type' => 'Person',
-                        'identifier' => pc['contributor_id']
+                        'identifier' => pc['contributor_id'],
+                        'contrib_type' => contributor_types[pc['contributor_id']]
```

Comment:
`contrib_type` isn't on the standard `Person`. It would be ideal if you could move them into one them from `contributors` to something like `author`, `illustrator`, or `editor` from https://schema.org/Book
If you can't, you could maybe use the `jobTitle` if that makes sense, or if it's acumen specific, then it should go in an `acumenAttributes` property.

Reply:
@Ken-Michalak I didn't realize these were modeled after schema.org types.

Given that ACUMEN has user-entered values, I think `acumenAttributes` makes the most sense (IP uses things like `by (Author)`.  We can use a JS Agent to map those to universal values like `author`.